tests/prefetch.ts.out

```diff
@@ -0,0 +1,10 @@
+Compiling [WILDCARD]tests/prefetch.ts
+Downloading http://localhost:4545/tests/subdir/subdir3/mod1.ts...
+Downloading http://localhost:4545/tests/subdir/subdir3/mod2.ts...
+Downloading http://localhost:4545/tests/subdir/subdir3/mod3.ts...
+Downloading http://localhost:4545/tests/subdir/subdir3/mod1.ts...
```

Comment:
Hrm. I guess this is #1496

Reply:
They should have the same underlying problem...